compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/model/SymbolTable.java

```diff
@@ -394,6 +394,7 @@ public void loadPredeclaredModules() {
                                                 Map.entry(Names.INT, this.langIntModuleSymbol),
                                                 Map.entry(Names.MAP, this.langMapModuleSymbol),
                                                 Map.entry(Names.OBJECT, this.langObjectModuleSymbol),
+                                                Map.entry(Names.RUNTIME, this.langInternalModuleSymbol),
```

Comment:
Similarly, why do we add a map to `langInternalModuleSymbol` instead of `langRuntimeModuleSymbol`?